user/conditional-builds-stages-jobs.md

```diff
@@ -46,10 +50,44 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
-Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions.
+Jobs need to be listed explicitly, i.e., using `jobs.include` (or its alias `matrix.include`), in order to specify conditions for them. Jobs created via [matrix expansion](/user/customizing-the-build/#build-matrix) currently cannot have conditions, but they can be conditionally excluded (see the next section).
 
 Jobs that do not match the condition will be skipped silently.
 
+## Conditionally Excluding Jobs
+
+Configure Travis CI to exclude jobs only when certain conditions are met.
+
```

Comment:
There is no need to start a new paragraph.

Reply:
done